projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html

```diff
@@ -28,6 +28,6 @@
 
 <ng-template #pageFooter>
     <mat-toolbar color="primary" class="footer-custom">
-        <span>NETGRIF, s.r.o. &copy; 2022</span>
+        <!-- TODO -->
```

Comment:
Same as above

Reply:
Depends on https://github.com/netgrif/components/pull/94

Reply:
placeholder of footer should be there